jobio-back/utils/queries.js

```diff
@@ -0,0 +1,25 @@
+
+const custom_queries = {
+    getOrgNames: 'SELECT title, unique_id FROM organization',
```

Comment:
have you considered using an ORM instead of having to write raw sql queries like this.